server.py

```diff
@@ -174,11 +187,16 @@ def do_save_response():
 
     response_type = str(survey_response.get('type'))
 
+    result = {'tx_id': survey_response.get('tx_id'),
+              'is_feedback': False}
```

Comment:
I think "is_feedback" would be better as just "feedback" as its a boolean value anyway